torch/functional.py

```diff
@@ -365,7 +365,11 @@ def meshgrid(*tensors, indexing: Optional[str] = None) -> Tuple[Tensor, ...]:
                 treated as tensors of size :math:`(1,)` automatically
 
             indexing: (str, optional): the indexing mode requested.
-                Only "ij" is currently supported.
+                Only "ij" and "xy" are currently supported. If "xy" is
```

Comment:
If we don't plan to support additional modes then I would state this positively as something like, "the indexing mode, either 'ij' or 'xy'. If 'xy'..."

Reply:
done